sdk/storage/azure-storage-blob-cryptography/src/main/java/com/azure/storage/blob/specialized/cryptography/BlobDecryptionPolicy.java

```diff
@@ -215,11 +205,10 @@
                             }
                         }
                         totalInputBytes.addAndGet(bytesToInput);
-                        plaintextByteBuffer.position(0); // Reset the position after writing.
 
                         // This is to get the decryptedBytes value into the next map to adjust the position of the
                         // output buffer because the data range was increased for decryption.
-                        plaintextByteBuffer.limit(decryptedBytes);
```

Comment:
The decryptedBytes is useless in current impl

Reply:
Is it better to set limit to decryptedBytes, and set pos to 0? Why leave some padding?

Reply:
Updated comment.